src/init/omp.ps1

```diff
@@ -45,7 +45,7 @@ function global:Set-PoshGitStatus {
     $config = $global:PoshSettings.Theme
     $cleanPWD = $PWD.ProviderPath.TrimEnd("\")
     $cleanPSWD = $PWD.ToString().TrimEnd("\")
-    $standardOut = @(&$omp --error="$errorCode" --pwd="$cleanPWD" --pswd="$cleanPSWD" --execution-time="$executionTime" --config="$config")
+    $standardOut = @(&$omp "--config=$config" "--error=$errorCode" "--pwd=$cleanPWD" "--pswd=$cleanPSWD" "--execution-time=$executionTime" 2>&1)
```

Comment:
what's the advantage of `"--config=$config"` over `--config="$config"`?

Reply:
rebasing issue.